### PR TITLE
fix(chore): Replace empty body status code with 200

### PR DESCRIPTION
**Pull request by yehor-manzhula**
## Motivation and Context

Empty response body status code 204 doesn't match internal specification
## How Has This Been Tested?
Updated tests

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Review by b1ff — APPROVED** [review]

File: src/server.ts [hunk 1]
```diff
@@ -269,7 +269,7 @@ export class InversifyExpressServer {
                     value();
                 } else if (!res.headersSent) {
                     if (value === undefined) {
-                        res.status(204);
+                        res.status(200);
```
Comment (blackrabbit99) [inline src/server.ts:272]: do not we have http statuses object in this library ?

Comment (blackrabbit99) [inline src/server.ts:272]: do not we have http statuses object in this library ?

**Review by blackrabbit99 — CHANGES_REQUESTED** [review]

File: src/server.ts (hunk 1 above)
Comment (yehor-manzhula) [reply src/server.ts:272]: No we have not.

Comment (yehor-manzhula) [reply src/server.ts:272]: No we have not.

**Review by blackrabbit99 — APPROVED** [review]

**Pull request merged**